CONTRIBUTOR_LADDER.md

```diff
@@ -36,18 +36,20 @@ An Organization Member must meet the responsibilities and has the requirements o
     * Work to accomplish the tasks they volunteer to do within the project
 * Qualifications:
     * Must have successful contributions to the project, including at least one of the following:
-        * Written and/or reviewed PRs,
-        * Made contributions that resolved Issues,
+        * Accepted and/or reviewed PRs,
```

Comment:
Do you mean "Have PRs merged and / or reviewed PRs"?

Reply:
@shizhMSFT Yes, this is what I mean.